Essentials/src/com/earth2me/essentials/User.java

```diff
@@ -199,6 +196,8 @@ public void payUser(final User reciever, final BigDecimal value, UserBalanceUpda
             reciever.setMoney(reciever.getMoney().add(value), cause);
             sendMessage(tl("moneySentTo", NumberUtil.displayCurrency(value, ess), reciever.getDisplayName()));
             reciever.sendMessage(tl("moneyRecievedFrom", NumberUtil.displayCurrency(value, ess), getDisplayName()));
+            TransactionEvent transactionEvent = new TransactionEvent(this.getSource(), reciever, value);
+            ess.getServer().getPluginManager().callEvent(transactionEvent);
```

Comment:
It's "receiver"

Reply:
No it's not

Reply:
To clarify: it's spelt that way in the code, not necessarily the proper spelling. I'm not here to fix typos.

Reply:
My inner perfectionist couldn't resist.

Reply:
@l1ttleO Feel free to make a separate PR to fix the typos if you feel the need, but it's not in scope for this one.